core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/tenant/JDBCTenantManager.java

```diff
@@ -1201,4 +1249,24 @@ private boolean checkUniqueIdColumnInTable() throws UserStoreException {
         }
         return false;
     }
+
+    /**
+     * Execute deletion queries.
+     *
+     * @param conn DB connection
+     * @param query SQL query
+     * @param tenantId Id of the tenant
+     * @throws Exception
```

Comment:
Add the specific Exception type. Also, the param descriptions should start with a capital and end with a full stop. Fix all the other comments accrodingly.

Reply:
Fixed in #2894